src/js/compat.ts

```diff
@@ -72,6 +78,24 @@ export async function initNodeModules() {
   };
 }
 
+/**
+ * If we're in Deno, we initialize the required modules.
+ * Otherwise, this does nothing.
+ * @private
+ */
+export async function initDenoModules() {
+  if (!IN_DENO) {
+    return;
+  }
+  // Workaround for https://github.com/emscripten-core/emscripten/issues/6241
+
+  let XHRModule = await import(
+    // @ts-ignore
+    /* webpackIgnore: true */ "https://deno.land/x/xmlhttprequest_deno@v0.0.2/mod.js"
```

Comment:
This should never be used with webpack, since deno has its own bundler from what I understand.
That package was uploaded by josephrocca, see https://deno.land/x/xmlhttprequest_deno@v0.0.2

Of course, the ideal solution would be to fix it in https://github.com/emscripten-core/emscripten/issues/6241 but that would require quite a bit of refactoring. So this is temporary workaround.